### PR TITLE
🌈 add feedback after mark as done a task

### LINKED ISSUE #8: show feedback after mask as done a task
After finishing a task, list the pending ones

### PR DESCRIPTION
**Pull request by brunocerecetto**
Closes #8

### CODE REVIEW

**Comment by toti1212** [comment]
Hi @brunocerecetto! Good to see you here! Thanks for work on this! 🙌.
Looks good for me! Could you make one minor changes?

File: src/commands/done.js [hunk 1]
```diff
@@ -16,6 +17,12 @@ class DoneCommand extends Command {
         data.todo.splice(args.INDEX - 1, 1);
         data.done.push(item);
         storage.write(data);
+        
+        this.log(chalk.green(CONSTANTS.DONE_HAS_BEEN_MARKED(args.INDEX)));
+        
+        // const dataTodo = await list('todo');
```
Comment (toti1212) [inline src/commands/done.js]: Could you remove these comments?

File: src/utils/constants.js [hunk 2]
```diff
@@ -13,8 +13,10 @@ module.exports = {
   // done
   DONE_INCORRECT_INDEX: index =>
     `⚠️  Does not exists an item with ID '${index}'`,
+  DONE_HAS_BEEN_MARKED: index =>
```
Comment (toti1212) [inline src/utils/constants.js]: Rename the constants. Maybe to `DONE_ITEM_DONE` or `DONE_ITEM_MARKED`

**Review by toti1212 — CHANGES_REQUESTED** [review]
Hi @brunocerecetto! Good to see you here! Thanks for work on this! 🙌.
Looks good for me! Could you make one minor change?

**Comment by brunocerecetto** [comment]
Done!

**Review by toti1212 — APPROVED** [review]

**Pull request merged**